lms/envs/production.py

```diff
@@ -973,3 +973,62 @@ def get_env_setting(setting):
                                                       COMPLETION_VIDEO_COMPLETE_PERCENTAGE)
 COMPLETION_VIDEO_COMPLETE_PERCENTAGE = ENV_TOKENS.get('COMPLETION_BY_VIEWING_DELAY_MS',
                                                       COMPLETION_BY_VIEWING_DELAY_MS)
+
+######################## CELERY ROUTING ########################
+
+# Defines alternate environment tasks, as a dict of form { task_name: alternate_queue }
+ALTERNATE_ENV_TASKS = {}
+
+# Defines the task -> alternate worker queue to be used when routing.
+EXPLICIT_QUEUES = {
```

Comment:
Does this mean that all tasks defined in any installed app have to be explicitly stated here or they will fall to the default queue?  Is that the behavior we want?  Is this going to be overridden on a per environment basis for edx deployments?

Reply:
Yes in this implementation the task should be mentioned here if it has to be routed to a queue other than the default one, @jmbowman  can answer that better.

Reply:
Tasks have always gone to the default queue unless specified otherwise on a task-by-task basis.  The change here is that we've moved the declaration of such exceptions from the task definitions to the Django settings, which should make it a bit easier both to understand where they go and to make changes to the routing (both for ourselves and Open edX operators).

Reply:
@feanil any concerns? we are ready to merge this PR.

Reply:
No concerns.  Thank you for the explanation.